src/mono/wasm/runtime/invoke-js.ts

```diff
@@ -309,22 +309,22 @@ function mono_wasm_lookup_function(function_name: string, js_module_name: string
 }
 
 export function set_property(self: any, name: string, value: any): void {
-    mono_assert(self, "Null reference");
+    mono_check(self, "Null reference");
```

Comment:
similarly i understand why mono_assert was added to the imports list here, but it's weird that mono_check is used without being defined